README.md

```diff
@@ -171,6 +186,39 @@ CONFIG:
   pooling_api: http://vmpooler.delivery.puppetlabs.net/
 ```
 
+### Custom hypervisor
+
+The following example shows one way of generating a custom hypervisor that
+includes both per-host configuration and global configuration.
+
+The term "custom" in this case signifies that it's not a built-in hypervisor
+(like `vmpooler`), which means we'll have to provide all the configuration
+ourselves as there isn't any built-in configuration for our hypervisor.
+
+```
+$ beaker-hostgenerator --hypervisor=custom --global={custom_api=http://api.custom.net} centos6-64{hypervisor=custom}
```

Comment:
Defining `custom` with both a global flag and a host specific hash may lead users to infer that both are required. Should we omit the global flag or set the global flag to a different value to illustrate the disparate effect of these two entries?